com.unity.render-pipelines.high-definition/Documentation~/HDRP-Asset.md

```diff
@@ -79,6 +79,10 @@ These settings control the draw distance and resolution of the decals atlas that
 | **Property**                    | **Description**                                              |
 | ------------------------------- | ------------------------------------------------------------ |
 | **Enable**                      | Enable the checkbox to make HDRP support dynamic resolution in your Unity Project. |
+| **- Enable DLSS**               | Enable the checkbox to make HDRP support NVIDIA Deep Learning Super Sampling (DLSS).<br/>This property only appears if you enable the NVIDIA package (com.unity.modules.nvidia) in your Unity project. |
+| **-- Performance Quality**      | Use the drop-down to select which performance mode DLSS operates on. The options are:<br/>&#8226; **Balanced**: - Balances performance with quality.<br/>&#8226; **MaxPerf**: - Fast performance, lower quality.<br/>&#8226; **MaxQuality**: - High quality, lower performance.<br/>&#8226; **UltraPerformance**: - Fastest performance, lowest quality.<br/>&#8226; **UltraQuality**: - Highest quality, lowest performance. |
```

Comment:
> | **-- Performance Quality**      |

I Believe this has change to "mode" now, to better match what we're doing in raytracing perf/quality mode